A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_InitPage.js

```diff
@@ -198,7 +198,7 @@ class CDUInitPage {
         mcdu.onRightInput[0] = async () => {
             let value = mcdu.inOut;
             mcdu.clearUserInput();
-            if (await mcdu.trySetZeroFuelWeightZFWCG(value)) {
```

Comment:
`trySetZeroFuelWeightZFWCG` is called from two places, should both places use your new function?